### PR TITLE
Use stock tarsnaper and simplify config

### PR DESCRIPTION
**Pull request by jbzdak**
Testing instructions: 
1. Deploy new instance of the pruner (can be in DEV project in Openstack ). Instructions are in Readme. Please use Xenial as the base image. 
2. Login to the instance in hosts. 
3. List all archives for some of the machines 
4. Execute dry-run of the expiry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,14 +16,15 @@ How to add new backup to be pruned
...
+4. Add new entry to `TARSNAPPER_JOBS`
 
 How to prune backups
 --------------------
 
 1. Login to the instance 
-2. `sudo prune-{{backup.name}}.sh`, for example  
+2. Pruning scripts are named `tarsnap-{{job.name}}.sh`
+3. Following operations are supported: 
+   a. List archives `sudo tarsnap-job.sh list`
```
Comment (omarkhan) [inline README.md]: I tried running this verbatim and it failed:

```
tarsnap-job.sh: command not found
```

Then I realised that the `-job` part needs to be replaced with the job name. Maybe add the `{{job}}` curly braces to make it clearer?

File: templates/run-tarsnapper.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+KEY_FILE="/etc/tarsnap/{{ item.name }}.key"
+CACHE_DIR="/var/cache/tarsnap/{{ item.name }}"
+
+/usr/local/bin/tarsnap --fsck --keyfile ${KEY_FILE} \
+ --cachedir ${CACHE_DIR} && \
+tarsnapper \
+ -o keyfile ${KEY_FILE} -o cachedir  ${CACHE_DIR} \
```
Comment (omarkhan) [inline templates/run-tarsnapper.sh:9]: Does stock tarsnapper support these options? They don't appear if I run `tarsnapper --help`

Comment (omarkhan) [inline templates/run-tarsnapper.sh:9]: Oh I get it it's passing options through to tarsnap. Never mind

**Comment by omarkhan** [comment]
@jbzdak tested and working. 👍

File: README.md (hunk 1 above)
Comment (jbzdak) [inline README.md]: Addressed.

**Pull request merged**